backend/src/routes/post.ts

```diff
@@ -44,4 +44,22 @@ post.get("/me", async (req: RequestWUser, res, next) => {
   res.send(posts);
 });
 
+post.get("/following", async (req: RequestWUser, res, next) => {
+  const { user } = req;
+  const { limit, skip } = req.query;
+
+  if (
```

Comment:
I think I saw this code in another PR used for other pagination things, if it is used more than once you can extract it to a utils function